### PR TITLE
Ben Tsao

### PR DESCRIPTION
**Pull request by cbtsao47**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1 +1,73 @@
...
+})
+
+const contentSec = document.querySelectorAll('.home .content-section');
+contentSec[0].addEventListener('mouseout', function(el) {
+    TweenMax.to(contentSec[0], 2, {rotation:720});
+    event.target.style.background = 'lightblue';
+})
+
+contentSec[1].addEventListener('mouseleave', function(el) {
+    TweenMax.to(contentSec[1], 2, {rotation:-720});
```
Comment (szincone) [inline js/index.js]: Good job on implementing greensock.

File: js/index.js [hunk 2]
```diff
@@ -1 +1,73 @@
-// Your code goes here
+const funBus = document.querySelector('.logo-heading');
+funBus.addEventListener('mouseover', function(event){
+    event.target.style.color = 'red';
+    event.target.style.fontSize = '6rem';
+});
+
+const navItems = document.querySelectorAll('.nav-link');
+navItems.forEach(function(item) {
```
Comment (szincone) [inline js/index.js]: Nice nesting and code-formatting.

File: js/index.js [hunk 3]
```diff
@@ -1 +1,73 @@
...
+    item.addEventListener('click',(item) => { item.preventDefault()});
+    item.addEventListener('mouseenter', function() {
+        console.log(event);
+        event.target.style.color = 'white';
+        event.target.style.background = 'red';
+        event.target.style.fontSize = '2.4rem';
+        event.target.style.padding = '10px 5px';
+    });  
+});
+
```
Comment (szincone) [inline js/index.js]: Maybe start leaving comments to explain your code, like you could add a comment here to explain what the event is and what it does.

File: js/index.js [hunk 4]
```diff
@@ -1 +1,73 @@
...
+destinations.forEach(function(el) {
+    el.addEventListener('click', function() {
+        TweenLite.to(destinations, 2, { ease: Bounce.easeOut, x: 200, rotation: 360});
+    })
+    
+})
+
+const button = document.querySelector('.home .destination .btn');
+button.addEventListener('click', function(event) {
+    event.stopPropagation();
```
Comment (szincone) [inline js/index.js]: Good use of stopPropagation to make sure this event doesn't fire the other clicks, when clicked.

File: stretch-assignment/index.html [hunk 5]
```diff
@@ -3,16 +3,31 @@
...
   <body>
     <script defer="defer" src="./index.js"></script>
-
-    <div class="blocks">
-      <div class="block block--red"></div>
-      <div class="block block--blue"></div>
-      <div class="block block--green"></div>
-      <div class="block block--pink"></div>
-      <div class="block block--gray"></div>
+    <h1>Rocket Nyan Cat Game</h1>
```
Comment (szincone) [inline stretch-assignment/index.html]: This was fun w the nyan cats flying off.

File: stretch-assignment/index.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+//   * When the mouse button is released, it should travel back to its original position.
+//   * A mouse down that occurs during a transition back to the original position should send it traveling to the right again  from it's current position.
+  
+//   You will be using `window.setInterval`. For this method, and any of those listed in the "Topics" section of this README, you are encouraged to use MDN as your resource for Documentation. MDN can be challenging to consume at first, so you may also use something like W3schools if the MDN documentation on a given subject is a bit too complicated. -->
+
+// 
+
+
+let counter = 0;
+// cave-cat way
```
Comment (szincone) [inline stretch-assignment/index.js]: Good job on implementing and conquering stretch.

File: stretch-assignment/index.html (hunk 5 above)
Comment (cbtsao47) [reply stretch-assignment/index.html]: thanks 😂 
Do you have any idea how to make it so the cats only are animated when the block is pressed?
like a jetpack kind of thing. I don't know how to make the event continuously fire. Looked into setInterval, but could not understand it via MDN and some youtube videos

**Pull request closed without merging**